client/ayon_core/plugins/publish/validate_editorial_asset_name.py

```diff
@@ -35,7 +35,7 @@ def process(self, context):
 
         existing_folder_paths = {
             folder_entity["path"]: (
-                folder_entity["path"].lstrip("/").rsplit("/")[0]
+                folder_entity["path"].lstrip("/").split("/")[:-1]
```

Comment:
This doesn't seem right. This will now return a list but I wouldn't  expact `folder_entity["path"]` value to be a list.

What are you describing to do here?

This might also benefit from better variable naming. What is `existing_folder_paths` if it's a dictionary? What is the key, and what is the value supposed to be?

Reply:
I have added some comments in code to describe required value

Reply:
We just realized that the plugin is not needed in AYON at all. There is no need to validate unique folder name across project in AYON.

Reply:
If the values are just the list of parent folder **names** (not their paths) then please update the `existing_folder_paths` variable name to describe what it contains, e.g. `folder_parent_names` or alike.